rfc.md

```diff
@@ -0,0 +1,58 @@
+# Friends Toy
+
+## Purpose
+This is a simple toy app designed to increase my familiarity with scala. 
+The basic purpose is to model friendships between people. 
+
+## Description
+For now all people will be held in a single "table" and will be self-joined in order to model relationships.
```

Comment:
What would the DDL looks like? How would a self-join accomplish our goals?

Reply:
What is a DDL?